<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalFilters</class>
 <widget class="QMainWindow" name="DigitalFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(64, 64, 64);
}

QTabWidget::pane { 
   border: 1px solid rgb(166, 166, 166);
}

QTabBar::tab {
  	background-color: rgb(77, 77, 77);
 }

 QTabBar::tab:selected {
  	background-color: rgb(140, 140, 140);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:selected:hover {
  	background-color: rgb(140, 140, 140);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:hover {
  	background-color: rgb(109, 109, 109);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:middle {
	border-right: 1px dashed rgb(166, 166, 166);
	border-left: 1px dashed rgb(166, 166, 166);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:last {
	border-top-right-radius: 10px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:first {
	border-top-left-radius: 10px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }


QComboBox QAbstractItemView {
    background-color: rgb(77, 77, 77);
}

QComboBox QAbstractItemView::item:focus{
    background-color: rgb(140, 140, 140);
}

QDoubleSpinBox{
	background-color: rgb(77, 77, 77);
	
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}

QDoubleSpinBox::up-button{
    image: url(:/imgs/imgs/drop_up_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::up-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::up-button:hover{
	background-color: rgb(0, 50, 0);
}

QDoubleSpinBox::down-button{
    image: url(:/imgs/imgs/drop_down_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::down-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::down-button:hover{
	background-color: rgb(0, 50, 0);
}

QWidget#centralwidget{
	background-color: rgb(64, 64, 64);
}

QWidget{
	color: rgb(255, 255, 255);
}

QComboBox{
	background-color: rgb(77, 77, 77);
	
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);

}


QComboBox::drop-down{
	image: url(:/sim/drop_down_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 2px solid rgb(127, 167, 127);
	border-left: 2px solid rgb(127, 167, 127);

	border-bottom: 2px solid rgb(0, 0, 0);
	border-right: 2px solid rgb(0, 0, 0);
}

QComboBox::drop-down:hover{
	background-color: rgb(0, 50, 0);
}

QComboBox::drop-down:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QPushButton{
	background-color: rgb(0, 79, 0);

	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);

	
	font: 12pt &quot;Helvetica&quot;;
	text-align:center;
}

QWidget{
	font: 12pt &quot;Helvetica&quot;;
}

QPushButton:hover{
	background-color: rgb(0, 50, 0);
}

QPushButton:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QLineEdit{
	background-color: rgb(77, 77, 77);
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}

QRadioButton::indicator{
	border-radius: 6px;
	border-top: 1.5px solid rgb(0, 0, 0);
	border-left: 1.5px solid rgb(0, 0, 0);

	border-bottom: 1.5px solid rgb(160, 160, 160);
	border-right: 1.5px solid rgb(160, 160, 160);
}

QRadioButton::indicator::checked{
	background-color: white;
}

QRadioButton::indicator::unchecked:hover{
	background-color: #9F9F9F;
}

QRadioButton::indicator::pressed{
	border: 1.5px solid #505050
}

QPushButton#reload_devices{
	image: url(:/imgs/imgs/reload.png);
	width: 11px;
	background-color: rgb(0, 79, 0);

	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);

	
	font: 12pt &quot;Helvetica&quot;;
	text-align:center;
}

QPushButton#reload_devices:hover{
	background-color: rgb(0, 50, 0);
}

QPushButton#reload_devices:pressed{
	border: 2px solid rgb(255, 255, 255);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="0" alignment="Qt::AlignBottom">
        <widget class="QWidget" name="bottom" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="filterButton">
            <property name="text">
             <string>FILTER</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeButton">
            <property name="text">
             <string>CLOSE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="top" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QLabel" name="title">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>30</y>
            <width>171</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>-1</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;concordo&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30px</string>
          </property>
          <property name="text">
           <string>Digital Filters</string>
          </property>
         </widget>
         <widget class="QLabel" name="logo">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>101</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/sim/logo.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>imgs/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Select your arduino board:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please, select your arduino board&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="mid" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Select your filter:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="iirButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>10</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Infinite Impulse Response&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>IIR</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="firButton">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Finitive Impulse Response&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>FIR</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imgs/recursos.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
